docs/admin/troubleshooting_updates.rst

```diff
@@ -104,8 +104,50 @@ Note that ``dom0`` and ``apply_dom0`` are separate steps.
    for assistance.
 
 4. Reboot the system. ``dom0`` updates are often
-   security-sensitive, and may requrie a reboot to take
+   security-sensitive, and may require a reboot to take
    effect.
+   
+GPG check failures
+^^^^^^^^^^^^^^^^^^
+
+If at any point you see a message that the GPG check failed,
+and you see an indication that a key has expired, your system
+may be trying to use an old copy of the SecureDrop Release Signing
+Key. 
+
```

Comment:
+1 Kev's suggestion about being specific. I would also suggest replacing the line that starts with "your system may be....." and just say "If you encounter <specific scenario>, reimport the SecureDrop Release Signing Key, which is locally available on your machine, by following the steps below."  or something. 

reasoning:
- "may be trying to" makes it sound like we don't know the root cause, but we do, or that the behaviour is inconsistent
- mentioning that the good key is available already on their machine in order to clarify that we aren't instructing sometimes users perform some workaround, and that the security of their system is not in doubt.

Reply:
That's a great suggestion, I just added a commit to implement your suggestion. Please let me know if you think it's sufficient, or if you'd rather go with your wording verbatim.